tests/hmm/hmumu_utils.py

```diff
@@ -2026,8 +2063,12 @@ def get_gen_sumweights(filenames):
     for fi in filenames:
         ff = uproot.open(fi)
         bl = ff.get("Runs")
-        arr = bl.array("genEventSumw")
-        arr2 = bl.array("genEventSumw2")
+        try:
+            arr = bl.array("genEventSumw")
+            arr2 = bl.array("genEventSumw2")
+        except KeyError:
+            arr = bl.array("genEventSumw_")
```

Comment:
might be good to put a comment here for future reference, perhaps also a printout.

Reply:
added.